README.md

```diff
@@ -50,14 +50,23 @@ Copy the template. Rename to include the date of intended publication in the fil
 
 ## Send the email
 
-**Create the campaign** using the [This Week in Tessel template](http://templates.mailchimp.com/getting-started/using-mailchimp/).
+**Create a new campaign**. Set the campaign and subject to the same name, e.g. "This Week in Tessel: *Some content here...*"
 
-Name the campaign something like "This Week in Tessel - Date"
+![Select "Saved Templates" and click the "TWIT Reference".](https://cloud.githubusercontent.com/assets/80639/18758377/d995a3e6-80c5-11e6-8008-a52d43aaa7bd.png) You'll be brought to the responsive text editor.
 
-Copy and paste the rendered markdown from any markdown renderer, instead of using Mailchimp's specialized formatting (because, what a pain otherwise). Mailchimp handles this pretty gracefully, but read it over anyway just in case.
+The main text block has an HTML editor (the leftmost icon with angle brackets `<>` in it). The easiest way to convert the Markdown content for this email into HTML for the email is to use the `twitgen` NPM module, included in this repository.
```

Comment:
is it useful to mention that the module is in this directory? It gave me pause because I wasn't sure if I was supposed to do something with that information. But it does seem worth mentioning somewhere - maybe as an aside?